lib/generators/react_on_rails/templates/base/base/client/webpack.config.js

```diff
@@ -5,9 +5,9 @@
 const webpack = require('webpack');
 const { resolve } = require('path');
 const ManifestPlugin = require('webpack-manifest-plugin');
-const webpackConfigLoader = require('react-on-rails').default.webpackConfigLoader; // eslint-disable-line import/no-unresolved
-
-const { env, paths, publicPath } = webpackConfigLoader(resolve('..', 'config', 'webpack'));
+const webpackConfigLoader = require('react-on-rails/node_package/lib/webpackConfigLoader').default;
+const configPath = resolve('..', 'config', 'webpack');
+const { env, paths, publicPath } = webpackConfigLoader(configPath);
```

Comment:
> @alexfedoseev are you okay with making everyone modify their webpack configs to import env, paths, and publicPath from this react-on-rails lib for webpackConfigLoader instead of setting themselves manually as normal?

Sounds like unnecessary and leaky layer.